web-console/knikubevirt/expose-vm-as-a-service/expose-vm-as-a-service.md

```diff
@@ -44,5 +44,11 @@ The user would go through the same flow.
 The Service is ready for use, and the user can see it in the Services list.
 
 ### Services on the Overview tab
+With this placement the user can view what services are exposing a resource, if any. The user can also take actions with any service or create a new one
 
 ![Services on the Overview tab](img/Expose_010Services.png)
+
+
```

Comment:
Should we clarify what this state will be when there are no services shown?

Reply:
> Should we clarify what this state will be when there are no services shown?
So I placed the 'Empty state from PF4' in this section:
https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/empty-state#empty-state
I have to admit that this preview does not make that much sense to me. The fact that there is no action here highlights that it is weird to have a section with no actions in it.
Also, Also, I don't think the icon actually helps with 'Empty state' elements. Using an icon here may lead to icons that don't correlate with the text, as we see here. I'm not sure e can provide an icon for every kind of list. 
![1-vm-Overview services-section-empty state](https://user-images.githubusercontent.com/48942365/59602333-20108000-910f-11e9-96a9-59094ddf34cf.jpg)

Should we just move on with this or open that design for discussion?

Reply:
> Should we just move on with this or open that design for discussion?

I'd say we log it as a new issue so that this PR can move forward separately.

Reply:
+1 I'm ok with that, I think it's worth considering so let's create an issue that focuses on that.